.github/workflows/pull_request.yml

```diff
@@ -265,6 +265,37 @@ jobs:
           TBA_API_READ_KEY: TEST_KEY
       - name: Run Build
         run: npm --prefix pwa run build
+  pwa-e2e:
+    name: "[pwa] E2E Tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create dovenv
+        run: ./ops/pwa/create_dotenv.sh --env
+        env:
+          VITE_TBA_API_READ_KEY: ${{ secrets.TBA_API_READ_KEY }}
+
+      - name: Set up Node
+        uses: actions/setup-node@v4.0.3
+        with:
+          node-version: "20.16.0"
+          cache: "npm"
+
+      - name: Install Dependencies
+        working-directory: pwa
+        run: npm ci
+      
+      - name: Install Playwright Browsers
+        working-directory: pwa
+        run: npx playwright install --with-deps
```

Comment:
Why are these using npx instead of npm, when we're installing the dev dependency for playwright?

Reply:
Dunno - just followed docs: https://playwright.dev/docs/intro

Reply:
ok 🤷